### PR TITLE
announce n30000

### PR DESCRIPTION
**Pull request by glozow**
Simple one, since this is so late

### CODE REVIEW

**Review by stickies-v — COMMENTED** [review]
Quickest turnaround ever! Looks great, just some structural changes/rewording for clarity etc:

<details>
<summary>git diff</summary>

```diff
diff --git a/_posts/2024-05-01-#30000.md b/_posts/2024-05-01-#30000.md
index 1ec5a9e..8cbda8b 100644
--- a/_posts/2024-05-01-#30000.md
+++ b/_posts/2024-05-01-#30000.md
@@ -12,19 +12,20 @@ commit:
 
 ## Notes
 
-- The p2p code uses a [`TxOrphanage`](https://github.com/bitcoin/bitcoin/blob/842f7fdf786fcbbdf3df40522945813404f8a397/src/net_processing.cpp#L1079-L1081) to store transactions with missing inputs, or orphan
+- An orphan transaction is a transaction with missing inputs. The p2p code uses a [`TxOrphanage`](https://github.com/bitcoin/bitcoin/blob/842f7fdf786fcbbdf3df40522945813404f8a397/src/net_processing.cpp#L1079-L1081) to store orphan
   transactions, to be reconsidered later if/when its parent transaction(s) are submitted to mempool.
   There are two ways this can happen:
 
     - [`ProcessOrphanTx`](https://github.com/bitcoin/bitcoin/blob/842f7fdf786fcbbdf3df40522945813404f8a397/src/net_processing.cpp#L3366), which is called at the start of each `ProcessMessages`, pops orphans from a
-      [work set](https://github.com/bitcoin/bitcoin/blob/master/src/net_processing.cpp#L3366) that is updated whenever a parent transaction is [accepted to mempool](https://github.com/bitcoin/bitcoin/blob/842f7fdf786fcbbdf3df40522945813404f8a397/src/net_processing.cpp#L3238).
+      [work set](https://github.com/bitcoin/bitcoin/blob/842f7fdf786fcbbdf3df40522945813404f8a397/src/net_processing.cpp#L3366) that is [updated](https://github.com/bitcoin/bitcoin/blob/842f7fdf786fcbbdf3df40522945813404f8a397/src/net_processing.cpp#L3238) whenever a parent transaction is accepted to mempool.
 
-    - There are also two locations where a low-feerate parent is paired with its child in the
-      orphanage to be submitted together as a package: [when a transaction fails for being low feerate](https://github.com/bitcoin/bitcoin/blob/842f7fdf786fcbbdf3df40522945813404f8a397/src/net_processing.cpp#L4666-L4675)
-      and [when a low feerate parent is downloaded again](https://github.com/bitcoin/bitcoin/blob/842f7fdf786fcbbdf3df40522945813404f8a397/src/net_processing.cpp#L4546-L4557).
-      This "opportunistic 1p1c package submission" logic was added in PR #28970.
+    - When a low-feerate parent is paired with its child in the
+      orphanage to be submitted together as a package. This happens in two locations:
+      - [when a transaction fails for being low feerate](https://github.com/bitcoin/bitcoin/blob/842f7fdf786fcbbdf3df40522945813404f8a397/src/net_processing.cpp#L4666-L4675)
+      - [when a low feerate parent is downloaded again](https://github.com/bitcoin/bitcoin/blob/842f7fdf786fcbbdf3df40522945813404f8a397/src/net_processing.cpp#L4546-L4557)
+      This "opportunistic 1-parent-1-child (1p1c) package submission" logic was added in PR [#28970](https://github.com/bitcoin/bitcoin/pull/28970).
 
-- Orphans can be removed in a few different ways:
+- An orphan can be removed in a few different ways:
 
     - When it is [successfully submitted to mempool](https://github.com/bitcoin/bitcoin/blob/842f7fdf786fcbbdf3df40522945813404f8a397/src/net_processing.cpp#L3240).
 
@@ -70,7 +71,7 @@ in `AddTx`, [immediately fails](https://github.com/bitcoin/bitcoin/blob/842f7fdf
 
 1. Instead of allowing multiple transactions with the same txid (where we are obviously wasting some space on a version we will not accept), should we allow a transaction to replace an existing entry in the `TxOrphanage`? What would be the requirements for replacement?
 
-1. What places in the code do we check whether the orphanage contains a transaction? Is the query done by wtxid, txid, or both? (Hint: there are at least 5).
+1. Where in the code do we check whether the orphanage contains a transaction? Is the query done by wtxid, txid, or both? (Hint: there are at least 5).
 
 1. This PR removes the ability to query the orphanage by txid, since the `TxOrphanage` no longer has an index by txid. Is that okay, and why or why not?

``` 
</details>

**Comment by glozow** [comment]
Thanks @stickies-v! Addressed comments

File: _posts/2024-05-01-#30000.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+---
+
+## Notes
+
+- The p2p code uses a [`TxOrphanage`](https://github.com/bitcoin/bitcoin/blob/842f7fdf786fcbbdf3df40522945813404f8a397/src/net_processing.cpp#L1079-L1081) to store transactions with missing inputs, or orphan
+  transactions, to be reconsidered later if/when its parent transaction(s) are submitted to mempool.
+  There are two ways this can happen:
+
+    - [`ProcessOrphanTx`](https://github.com/bitcoin/bitcoin/blob/842f7fdf786fcbbdf3df40522945813404f8a397/src/net_processing.cpp#L3366), which is called at the start of each `ProcessMessages`, pops orphans from a
+      [work set](https://github.com/bitcoin/bitcoin/blob/master/src/net_processing.cpp#L3366) that is updated whenever a parent transaction is [accepted to mempool](https://github.com/bitcoin/bitcoin/blob/842f7fdf786fcbbdf3df40522945813404f8a397/src/net_processing.cpp#L3238).
```
[the remark this answers is not in the document]
Comment (glozow) [reply _posts/2024-05-01-#30000.md]: Fixed thanks

**Review by stickies-v — APPROVED** [review]
LGTM, thanks @glozow !

**Pull request merged**